### ISSUE TITLE
Update default button styles.

### ISSUE THREAD

**Issue by anniehirshman** [GetOrganizedBrooklyn/getorganizedbk-website#9]
Update button colors in the default styles, Home Page, and Working Group pages to match the following visual: https://www.figma.com/file/WHVnX8xXlIt3gd2r0rkR1zbk/GetOrganizedBK-Website?node-id=12%3A3

See Issue #4 

## Primary Buttons:

**Primary buttons (specs below) should be set as the default style** in Appearance > Customize > Buttons, so that future buttons/CTA modules automatically load with this style:

Text color: FFF (white)
Button color: 1A6A75 (teal)
Hover color: 433B4F (dark blue)

Any existing primary buttons with custom styles will need to be reverted to the default style (on all pages). 

## Home Page secondary buttons:

Secondary buttons on the home page (Pages > Home) for the working group names will then need to be overwritten with the following style: 

Text color: #433B4F (dark blue)
Hover color: #21212C (black)

## Tertiary buttons and links:

All tertiary buttons/links should have the following style: 

Text color: #1A6A75 (teal)
Hover color: #433B4F (dark blue)
